vsintegration/src/FSharp.Editor/Navigation/GoToDefinition.fs

```diff
@@ -109,7 +109,10 @@ module private ExternalSymbol =
         | _ -> []
 
 // TODO: Uncomment code when VS has a fix for updating the status bar.
```

Comment:
BTW do you think this is still relevant? :)

Reply:
😀 I bet VS statusbar works fine. Just not in our code. We should really stand of the shoulders of experts and either copy what [VSIX Community Toolkit](https://github.com/VsixCommunity/Community.VisualStudio.Toolkit) does or even just install their nuget.